src/conmon.c

```diff
@@ -312,7 +312,7 @@ int main(int argc, char *argv[])
 		.pid_to_handler = pid_to_handler,
 		.exit_status_cache = NULL,
 	};
-	int signal_fd = get_signal_descriptor(SIGCHLD);
+	int signal_fd = get_signal_descriptor(SIGCHLD, SIGUSR1);
```

Comment:
I think I would rather hardcode these in `get_signal_descriptor` then pass two signals arbitrarily

Reply:
we have two different versions of `get_signal_descriptor`.  If we want to move the list in `get_signal_descriptor` we would need a way to define them in a single place

Reply:
fixed in the last version